gui/src/2D/xmlLoader.ts

```diff
@@ -289,6 +289,11 @@ export class XmlLoader {
         return this._isLoaded;
     }
 
+    /** @deprecated Please use getNodeById instead */
+     public getNodeByID(id: string): any {

```

Comment:
Is that needed?

Reply:
you tell me :) I can remove it

Reply:
you just added it, no? there is no need to add new functions, just change existing ones

Reply:
yes you're right, I put myself in automatic mode and add a few unwanted deprecated function 👎 .